build.gradle.kts

```diff
@@ -0,0 +1,96 @@
+import no.skatteetaten.aurora.gradle.plugins.AuroraPlugin
+import org.jetbrains.kotlin.gradle.dsl.Coroutines
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.springframework.boot.gradle.tasks.bundling.BootJar
+
+plugins {
+    val kotlinVersion = "1.2.30"
+    kotlin("jvm") version kotlinVersion
+    kotlin("plugin.spring") version kotlinVersion
+    id("io.spring.dependency-management") version "1.0.4.RELEASE"
```

Comment:
We should probably use the technique described in 3.2. Using Spring Boot’s dependency management in isolation (https://docs.spring.io/spring-boot/docs/2.0.0.RELEASE/gradle-plugin/reference/html/) to avoid having to know the dependency-management plugin version.